Gaijinizer patch/Troops.txt

```diff
@@ -10667,10 +10678,10 @@ who was modified and created before#MTLed
  その暖かさと気持ち良さで張りつき干からびる人間を
 獣のオス以下とあざけ笑っておったわ。
 > CONTEXT: Troops/283/pages/0/43/Dialogue/enemy17
-Certainly that's when the guy hit a shelter in
-the snowy mountains. Humans who are stuck and
-dried with their warmth and comfort
-I laughed with the beast male and below.#MTLed
+If I recall, they were used to attack a shelter
+in some snowy mountains. She mocked and derided
+the humans who sought comfort and warmth as they
+were drained dry as lower than beasts.#RonIffy
 > END STRING
```

Comment:
Just some rewording:
>_**If I recall, they were used to attack a shelter
in some snowy mountains. She mocked and derided
the humans who sought comfort as lower then beasts
as they were drained completely dry.**_